tests/unit/__init__.py

```diff
@@ -1,4 +1,4 @@
 import os
 
-os.environ["DEPLOYMENT_STAGE"] = "test"
+os.environ["DEPLOYMENT_STAGE"] = "dev"
```

Comment:
This actually causes some of the authorizer tests not to run right? If you switch it away from "test"?

Reply:
Actually it will make them run every time, which we don't want until we fix the issues with running out of generated auth0 access tokens for machine to machine.

Reply:
Uhh yea this undesired. Deprecating this change - keeping as `test` in this PR. will need to reconcile with Trent's current work.